src/test/java/tests/TC02_UserCanNotLoginWithBlankUsername.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import helper.Constant;
+import models.Account;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.HomePage;
+import page_objects.LoginPage;
+
+public class TC02_UserCanNotLoginWithBlankUsername extends TestBase {
+    HomePage homePage = new HomePage();
+    LoginPage loginPage = new LoginPage();
+    String blank = "";
+
+    @Test(description = "User can't login with blank 'Username' textbox")
+    public void testCase02() {
+        Account account = new Account(blank, Constant.PASSWORD);
```

Comment:
you can enter `""` directly, we don't need a variable for just a simple blank

Reply:
@tanmle I have fixed my code, please review it again